chart/tugger/templates/admission-registration.yaml

```
@@ -13,7 +13,7 @@
   {{ $caCert = $ca.Cert }}
 {{- end }}
 
-{{- if .Values.createValidatingWebhook }}
+{{- if .Values.validatingWebhook.create}}
```

A better alternative would be to make it backwards compatible:
```suggestion
{{- if or .Values.createValidatingWebhook .Values.validatingWebhook.create}}
```

---

Sweet, I took your suggestion here! Thanks.